README.md

```diff
@@ -75,6 +75,10 @@ Resource types:
 - [Git Interactive](https://www.katacoda.com/courses/git) 🎮 🎁
 - [Pro Git Book](https://git-scm.com/book/en/v2) 📚 🎁
 
+### Exercises
+
+-[Simple projects using HTML and CSS](https://codeclubprojects.org/en-GB/webdev/)
```

Comment:
I think we can leave exercises out for now. And add it in later when we have curated a good number of challenges.